tests/components/template/test_alarm_control_panel.py

```diff
@@ -204,7 +204,7 @@ async def test_no_action_scripts(hass, start_ha):
                     "platform": "template",
                     "panels": {
                         "bad name here": {
-                            "value_template": "{{ disarmed }}",
+                            "value_template": "disarmed",
```

Comment:
Why is this cahnged?

Reply:
Because `{{ disarmed }}` is not a valid template. I tried it in my test instance config and got an error because `disarmed` is a variable, not a string in this context. I went with a fixed version in new tests and fixed existing ones while I'm at it. Do you think I should leave it as is?

Reply:
It's fine to change that here, it makes sense to have a more reasonable config in the tests 👍